src/schema/userSchema.ts

```diff
@@ -0,0 +1,101 @@
+import { Schema } from 'mongoose'
+
+const validateEmail = (email: string) => {
+    const re = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/
+    return re.test(email)
+}
+
+export const UserSchema: Schema = new Schema({
+    customer_key: {
+        type: String,
+        required: [true, `customer_key is missing.`],
+        unique: true
+    },
+    phone: {
+        type: String,
+        unique: true,
+        required: [true, `No phone number given.`]
+    },
+    gender: {
+        type: String,
+        required: [true, `No gender is given`]
+    },
+    orientation: {
+        type: String
+    },
+    location: {
+        type: {
+            type: String,
+            default: `Point`
+        },
+        coordinates: [Number]
+    },
+    is_active: {
+        type: Boolean,
+        default: 0
+    },
+    user_name: {
+        required: [true, `User name can't be empty`],
+        type: String,
+        trim: true,
+    },
+    blocked_list: [
+        Number
+    ],
+    profile_info: {
+        about: {
+            type: String
+        },
+        job_title: {
+            type: String
+        },
+        company: {
+            type: String
+        },
+        university: {
+            type: String,
+        },
+        live_in: {
+            type: String
+        },
+        profile_pic: [
```

Comment:
can we make it generaic?

Reply:
I think its would be better if we have a section for the upload of the media by a person and the profile pic section.